### PR TITLE
adding OKD related changes to release/3.4

### PR DESCRIPTION
**Pull request by sankarpn**
release/3.4 integration tests was not supporting OKD platform. These changes are made to run the integration tests in release/3.4 to run on OKD platform.

https://build.weblogick8s.org:8443/job/wko-okd-test/254/

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9970/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java [hunk 1]
```diff
@@ -12,9 +12,9 @@
 import com.gargoylesoftware.htmlunit.html.HtmlPage;
 import io.kubernetes.client.openapi.ApiException;
 import io.kubernetes.client.openapi.Configuration;
-import io.kubernetes.client.openapi.apis.ApiextensionsV1Api;
+import io.kubernetes.client.openapi.apis.ApiextensionsV1beta1Api;
```
Comment (rjeberhard) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java:15]: You are unintentionally reverting a change here to move from the "v1beta1" API's to the modern "v1" versions.

**Review by rjeberhard — CHANGES_REQUESTED** [review]
Restore the intentional changes to remove "v1beta1" API's.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java (hunk 1 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java]: Yes I remember this change :-) Thanks for catching. Fixed

**Comment by rjeberhard** [comment]
@sankarpn, thanks for fixing the v1beta1 issue so quickly... It looks like those 5 "code smells" from Sonar are valid. You should use assertEquals() in place of assertTrue(something == something else)

**Review by rjeberhard — APPROVED** [review]

**Pull request merged**